single-interviews.php

```diff
@@ -113,33 +121,41 @@
 				<div id="tables" class="content" style="display: none;">
 					<div class="tabcontent">
 						<ul class="tabnav">
-							<?php if ( $individuals_id ) echo '<li class="active"><a href="#tab1">Individuals<div></div></a></li>'; ?>
```

Comment:
eek! extra closed divs inside the a tag! we should make a note and clean that up in a separate branch.